src/cli/cli.cpp

```diff
@@ -2443,25 +2443,57 @@ template <> otError Interpreter::Process<Cmd("delaytimermin")>(Arg aArgs[])
 }
 #endif
 
+/**
+ * @cli detach
+ * @code
+ * detach
+ * Finished detaching
+ * Done
+ * @endcode
+ * @par
+ * Start the graceful detach process by first notifying other nodes (sending Address Release if acting as a router, or
+ * setting Child Timeout value to zero on parent if acting as a child) and then stopping Thread protocol operation.
+ * @sa otThreadDetachGracefully
+ */
 template <> otError Interpreter::Process<Cmd("detach")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
 
+    /**
+     * @cli detach async
```

Comment:
This one uses the otThreadDetachGracefully API too, why not `@sa` to it as well?

Reply:
Thanks @Vyrastas. New push contains all the changes.